experimental/dds/tree2/src/feature-libraries/editable-tree-2/lazyField.ts

```diff
@@ -278,6 +337,10 @@ export class LazySequence<TTypes extends AllowedTypes>
 	public get asArray(): readonly UnboxNodeUnion<TTypes>[] {
```

Comment:
if we want, we can combine map and asArray into one function simply by making the function you provide optional. This follows the pattern with Array.from.

That said I don't think that has great discoverability and I'm not sure what we would name it, so I'm not recommending it, just noting its possible.

Reply:
That's a good point. I actually like that idea a lot, and would prefer it over having `asArray` myself, but I agree that it wouldn't be discoverable.

Reply:
asArray has to exist to meet our enumerable own property rules actually, so we can't get rid of it. I forgot that was a property and not a method here.